modules/imgproc/src/morph.cpp

```diff
@@ -1905,6 +1907,22 @@ void cv::morphologyEx( InputArray _src, OutputArray _dst, int op,
         erode( temp, temp, kernel, anchor, iterations, borderType, borderValue );
         dst = temp - src;
         break;
+    case MORPH_HITMISS:
+        CV_Assert(src.type() == CV_8U && src.channels() == 1);
+        k1 = (kernel == 1) / 255;
+        k2 = (kernel == -1) / 255;
+        normalize(src, src, 0, 1, NORM_MINMAX);
```

Comment:
Original author normalized `src` to easily compute its complement for the second erosion (`erode(1-src,...)`). Maybe not the most elegant solution, so I'll remove that normalization and use a `bitwise_not`